app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if len(command) == 3 and command[1] != command[2]:
+        with open(command[1]) as file1:
+            file1 = file1.read()
+            file_empty = len(file1)
+            if file_empty != 0:
+                with open(command[2], "w") as file2:
```

Comment:
add more declarative variable names for command[1], command[2], file1 and file2
I don't even realize what is it